src/App.tsx

```diff
@@ -109,6 +209,7 @@ export const App: React.FC = () => {
               <div className="loader" />
             </div>
           </div>
+          {todosToShow.map(todo => <TodoItem key={todo.id} todo={todo} />)}
```

Comment:
also for correct structure of your project need to add TodoList component and inside that component you can map your TodoListItems